### PR TITLE
[NO-TICKET] Update link tester instructions

### PR DESCRIPTION
**Pull request by mjang-cobalt**
Note: with htmltest, I found an Alt Text issue with the images in the `Integrations/_index.md` file. We'll address that separately. The question for you, @ana-dashuk-cobalt , is whether these instructions work for you.

### Added

n/a

### Updated

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| [README section on htmltest] | [Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/pull/222/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L75) | [Procedure to install the htmltest tool] |

## Preview This Change

To see how this change looks in production, see the link to the README in the PR, starting with line 75

## Variables

n/a

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,23 +73,85 @@ After you've made your working copy of the site repository, from its root folder
...
+alt attribute missing --- integrations/index.html --> /integrations/Jira.png
+```
+
+With images, for accessibilty, we need to include "Alt Text". In this case, you might see Markdown code like:
+
+```
+![](/gsg/PentestFlowOverview.png)
+```
+
+<!-- NEED TO TEST
```
Comment (mjang-cobalt) [inline README.md:138]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -73,23 +73,85 @@ After you've made your working copy of the site repository, from its root folder
...
+To accommodate screen readers, we need "Alt Text" similar to: 
+
+```
+![UI Flow for Pentests](/gsg/PentestFlowOverview.png "UI Flow for Pentests")
+```
+-->
```
Comment (mjang-cobalt) [inline README.md:144]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -73,23 +73,83 @@ After you've made your working copy of the site repository, from its root folder
...
 
-After you fix broken links, run the following commands to rebuild content in
-the public/ subdirectory:
+### Install htmltest
+
+Follow the instructions to install `htmltest` in the /usr/local/bin folder. The current (1 Nov 2022)
+website suggests the following command, which works if you're using the `bash` shell:
+
+```
+curl https://htmltest.wjdp.uk | sudo bash -s -- -b /usr/local/bin
```
Comment (mjang-cobalt) [inline README.md:86]:
```suggestion
sudo curl https://htmltest.wjdp.uk | sudo bash -s -- -b /usr/local/bin
```

File: README.md [hunk 4]
```diff
@@ -73,23 +73,83 @@ After you've made your working copy of the site repository, from its root folder
...
+```
+
+If you're using the zsh shell (default in MacOS):
+
+```
+curl https://htmltest.wjdp.uk | sudo zsh -s -- -b /usr/local/bin
```
Comment (mjang-cobalt) [inline README.md:92]:
```suggestion
sudo curl https://htmltest.wjdp.uk | sudo zsh -s -- -b /usr/local/bin
```

File: README.md [hunk 5]
```diff
@@ -73,23 +73,83 @@ After you've made your working copy of the site repository, from its root folder
...
+website suggests the following command, which works if you're using the `bash` shell:
+
+```
+sudo curl https://htmltest.wjdp.uk | sudo bash -s -- -b /usr/local/bin
+```
+
+If you're using the zsh shell (default in MacOS):
+
+```
+sudo curl https://htmltest.wjdp.uk | sudo zsh -s -- -b /usr/local/bin
```
Comment (mjang-cobalt) [inline README.md:92]: @ana-dashuk-cobalt I think I found the problem. Users need to make sure that they're in `sudo` (admin) mode when installing `htmltest`. You should be able to:


cd /path/to/cobalt-product-public-docs
sudo curl https://htmltest.wjdp.uk | sudo zsh -s -- -b /usr/local/bin
(Here's a screenshot of the messages that should appear -- )

Screen Shot 2022-11-01 at 11 02 15 AM

hugo mod clean
hugo
htmltest

Comment (ana-dashuk-cobalt) [reply README.md:92]: @mjang-cobalt This instruction works for me, thank you!

**Review by ana-dashuk-cobalt — APPROVED** [review]

**Pull request merged**